finder-app/Makefile

```diff
@@ -0,0 +1,11 @@
+CC = gcc
+
+ifeq ($(CROSS_COMPILE), aarch64-none-linux-gnu-)
+	CC = aarch64-none-linux-gnu-gcc
+endif
```

Comment:
@abth5346 you can simplify the logic here and support other CROSS_COMPILE prefixes using `CC = $(CROSS_COMPLIE)gcc`